cmds/dhclient/dhclient.go

```diff
@@ -36,6 +36,7 @@ const (
 )
 
 var (
+	attempts     = flag.Int("attempts", 5, "Max number of attempts for client to receive a valid DHCP reply from server")
```

Comment:
I think that maybe this whole waiting for a reply thing should just have an overall timeout instead of any number of attempts.